package.json

```diff
@@ -35,20 +35,23 @@
   "devDependencies": {
     "@types/jest": "^28.0.0",
     "@types/react": "^17.0.11 || ^18.0",
-    "@types/react-dom": "^17.0.7|| ^18.0",
+    "@types/react-dom": "^17.0.7 || ^18.0",
```

Comment:
While you're at it, please remove the ^17 dependencies. If the dev deps needs at least react v18 then there is no reason to allow types for v17.